specs/commandline/sign.md

```diff
@@ -42,8 +42,8 @@ Flags:
        --plugin string              signing plugin name. This is mutually exclusive with the --key flag
        --plugin-config stringArray  {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values.
        --signature-format string    signature envelope format, options: "jws", "cose" (default "jws")
-       --tsa-root-cert string       filepath of timestamp authority root certificate
-       --tsa-url string             RFC3161 Timestamping Authority (TSA) server URL
+       --timestamp-root-cert string       filepath of timestamp authority root certificate
+       --timestamp-url string             RFC 3161 Timestamping Authority (TSA) server URL
   -u,  --username string            username for registry operations (default to $NOTATION_USERNAME if not specified)
```

Comment:
The rendering is broken: descriptions are not aligned.

Reply:
Ah, nice catch.